test/packages_test.rb

```diff
@@ -1016,4 +1016,85 @@ def product(properties = {})
       end
     end
   end
+
+  describe "#SelectProduct" do
+    before do
+      allow(subject).to receive(:Initialize).with(true)
+    end
+
+    context "when it is called in continue stage" do
+      before do
+        allow(Yast::Stage).to receive(:cont).and_return(true)
+      end
+
+      it "do not install any product" do
```

Comment:
Bad English: Use `does not` here. If in doubts it helps to read it as an English sentence: `it does not install ....`